src/main/java/seedu/address/model/ItemModelManager.java

```diff
@@ -234,4 +253,72 @@ public void clear() {
     public void sort() {
         this.visualList = visualList.sort();
     }
+
+    private void togglePriorityMode() {
+        priorityMode = !priorityMode;
+    }
+
+    /**
+     * Enable and disable the priority mode
+     * @return a boolean value. If true, means priority mode is on, else returns false.
+     * @throws IllegalListException if the visualList is not a task list.
+     */
+    public boolean togglePriority() throws IllegalListException {
```

Comment:
Do you want to merge this method with togglePriorityMode while retaining the method name togglePriorityMode?
There might be some confusion between togglePriority and changePriority.

Reply:
Ok!